sklearn/compose/_column_transformer.py

```diff
@@ -136,6 +136,12 @@ class ColumnTransformer(TransformerMixin, _BaseComposition):
         sparse matrix or a dense numpy array, which depends on the output
         of the individual transformers and the `sparse_threshold` keyword.
 
+    output_indices_ : dict
```

Comment:
I am unsure of the name since this is a map to slices.  Something like `output_slices_` does not sound that much better.

Reply:
Happy to change the name when / if there's a consensus. Reviewers don't seem to be of the same mind.

Reply:
> Reviewers don't seem to be of the same mind.

This is usually a beneficial since different reviewers see things the other did not. But when it comes to opinions, such as the naming of an attribute, it may makes things go longer. 

Thinking about this again, let's go with `output_indices_`.